medicine/run.py

```diff
@@ -188,4 +198,14 @@ def run_medicine(
             trainer=trainer,
         )
 
-    return trainer
+
+    if return_motion:
+        from spikeinterface.sortingcomponents.motion import Motion
+        motion = Motion(
+            displacement=[pred_motion],
+            temporal_bins_s=[time_bins],
+            spatial_bins_um=depth_bins,
+        )
+        return trainer, motion
```

Comment:
having this always could be more easy no ?
But I do not want to break some compatibility with you lab usage.